src/components/chatbox/ai-feature-toast.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState, useEffect } from 'react'
+import { IconWand24 } from '@hashicorp/flight-icons/svg-react/wand-24'
+
+import { toast, ToastColor } from 'components/toast'
+import useAuthentication from 'hooks/use-authentication'
+
+// TODO(kevinwang): see `make-dark-mode-notification.ts` for persistent-hiding
+export function AIFeatureToast() {
+	const [shown, setShown] = useState(false)
+	const { session } = useAuthentication()
+	useEffect(() => {
+		if (shown) {
+			return
+		}
+
+		if (session?.meta.isAIEnabled) {
+			toast({
+				color: ToastColor.highlight,
+				icon: <IconWand24 />,
+				title: 'Welcome to the Developer AI closed beta',
+				description: 'Try it out in our cmd+K menu!',
+				autoDismiss: 10000,
+			})
+			setShown(true)
+		}
+	}, [session?.meta.isAIEnabled, shown])
+	return null
+}
```

Comment:
### What?

Renders the following:
<img width="506" alt="CleanShot 2023-09-08 at 11 27 52@2x" src="https://github.com/hashicorp/dev-portal/assets/26389321/f89df693-b728-43a1-a07d-020c70011418">